models/topics.models.js

```diff
@@ -90,5 +90,19 @@ function updateArticles(article_id, incVotes){
     })
 }
 
+function removeComment(comment_id){
+    const query = `DELETE FROM comments
+                    WHERE comment_id = $1
+                    RETURNING *
+                   `
```

Comment:
You could change the name of your files here as it give the impression theses models (and controllers) only cover topic-related endpoints. You could either separate them out or give these files a more generic name like `app.models.js`